simple_message.py

```diff
@@ -1,6 +1,4 @@
-# The following script has been developed by Amartya Mondal.
-# For more details visit https://atm1504.in
-# Follow me at https://github.com/atm1504
+#Copyright@Souhardya & Amartya.
```

Comment:
Don't remove the other lines. Just add your name

Reply:
Okay.Chill